features/nfc/nfc/common/Mime.h

```diff
@@ -0,0 +1,136 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2018 ARM Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef NFC_COMMON_MIME_H_
+#define NFC_COMMON_MIME_H_
+
+#include "platform/Span.h"
+#include <nfc/ndef/RecordParser.h>
+#include <nfc/ndef/MessageBuilder.h>
+#include <stdint.h>
+
+namespace mbed {
+namespace nfc {
+namespace common {
```

Comment:
This is very NDEF-related, shouldn't it be in the NDEF namespace?

Reply:
Good point, should it be `ndef::common`  or just inside `ndef` ?